scripts/reboot

```diff
@@ -30,6 +30,36 @@ function clear_warm_boot()
     /sbin/kexec -u || /bin/true
 }
 
+SCRIPT=$0
+
+function show_help_and_exit()
+{
+    echo "Usage ${SCRIPT} [options]"
+    echo "    Request rebooting the device. Invoke platform tool when available."
+    echo "    This script will shutdown syncd before rebooting."
+    echo " "
+    echo "    Available options:"
+    echo "        -h -? : getting this help"
+
+    exit 0
+}
+
+function parse_options()
+{
+    while getopts "h?" opt; do
+        case ${opt} in
+            h )
+                show_help_and_exit
+                ;;
+            \? )
```

Comment:
Suggest combining these case statements as follows:
```
    case ${opt} in
        h|\?)
        show_help_and_exit
        ;;
    esac
```

Please test and confirm that it works as expected.

Reply:
Thanks! Done.